src/github.com/matrix-org/dendrite/cmd/dendrite-roomserver/main.go

```diff
@@ -43,10 +45,13 @@ func main() {
 	}
 
 	consumer := input.Consumer{
-		Consumer:             kafkaConsumer,
+		ContinualConsumer: common.ContinualConsumer{
```

Comment:
At present, I want to keep it like this. Once we've used `ContinualConsumer` a bit more and get a feel for how best to structure it, then this may change.